### ISSUE TITLE
bngblaster DHCPv6 reporting wrong sessions established

### ISSUE THREAD

**Issue by BSpendlove** [rtbrick/bngblaster#140]
HI there, thanks for the amazing tool you've released for free, been using it

**Describe the bug**

I still need to read into the RFCs to see if this is the expected behavior with dhcpv6 however dhcpv6 sessions are showing as completed when the dhcpv6 server sends a reply with a NoBinding (Option 13 - OPTION_STATUS_CODE, sub-option 3 "NoBinding" https://www.rfc-editor.org/rfc/rfc8415.html#section-21.13)

I was expecting the bngblaster to reattempt dhcpv6 if there is no binding but maybe I need to read up more on the RFC

**To Reproduce**

Setup any DHCPv6 server that would return a NoBinding in the v6 reply, specifically in my scenario I am authenticating v4 sessions on a Cisco ASR9k and CPE behavior is that it attempts dhcpv6 only after v4, I can't reproduce the same scenario on bngblaster because it seems to send both address families exactly at the same time however I'd at least expect a NoBinding to not officially think the dhcpv6 session is complete and reattempt it again

Version (`bngblaster -v`):
```
bngblaster -v
Version: 0.8.8
Compiler: GNU (11.3.0)
IO Modes: packet_mmap_raw (default), packet_mmap, raw
```

JSON configuration:
apologies, I've been trying different things to try to stagger the dhcp requests
```json
{
        "interfaces": {
                "access": {
                        "interface": "ens20",
                        "type": "ipoe",
                        "outer-vlan-min": 1000,
                        "outer-vlan-max": 1002,
                        "inner-vlan-min": 2,
                        "inner-vlan-max": 4094,
                        "ipv4": true,
                        "ipv6": true,
                        "dhcp": true,
                        "dhcpv6": true
                }
        },
        "ipoe": {
                "arp-timeout": 10
        },
        "dhcp": {
                "enable": true,
                "broadcast": true,
                "timeout": 20,
                "retry": 30,
                "release-interval": 60,
                "release-retry": 10
        },
        "dhcpv6": {
                "enable": true,
                "timeout": 30,
                "retry": 10,
                "access-line": true,
                "rapid-commit": true
        },
        "access-line": {
                "agent-remote-id": "bng-demo-{session-global}",
                "agent-circuit-id": "0.0.0.0/0.0.0.0 eth0:{session-global}",
                "rate-up": 1024,
                "rate-down": 16384
        }
}
```

1. Run the command `bngblaster -C config.json -I -c 1000`
2. DHCPv6 sessions will all complete upon receiving NoBinding in the dhcpv6 reply

**Expected behavior**

I expect the dhcpv6 to reattempt dhcpv6 process when receiving a NoBinding

**Screenshots**
bngblaster shows 4000 dhcpv6 sessions established (apologies example pictures are when I was testing 4k clients)
![image](https://user-images.githubusercontent.com/8687668/212335759-63991cb1-c32f-4b98-9a31-3668f55c81f5.png)

Dhcpv6 server returning NoBinding in the dhcpv6 advertise and also even the reply
![image](https://user-images.githubusercontent.com/8687668/212336115-b3c9fe2c-eb99-4bcb-b679-30f49b8aae35.png)

![image](https://user-images.githubusercontent.com/8687668/212335909-cf69ddb4-3f3d-44ae-be18-528e35908e87.png)

**Additional context**

I can provide packet captures if required with and without rapid-commit, similar behavior if I leave that feature enabled or not, thanks!

**Comment by GIC-de** [comment]
A capture file would be great! You can attach the capture here or send to bngblaster@rtbrick.com!

**Comment by GIC-de** [comment]
This issue should be fixed in latest `dev` branch (765e797). Could you please verify?

```
Jan 19 13:56:13.527643 DHCPv6 (ID: 1) Start DHCPv6
Jan 19 13:56:13.527647 DHCPv6 (ID: 1) DHCPv6-Solicit send
Jan 19 13:56:13.530254 DHCPv6 (ID: 1) DHCPv6-Reply received
Jan 19 13:56:13.530285 DHCPv6 (ID: 1) IA_PD received with status code 6 (NoPrefixAvail)
```

**Comment by BSpendlove** [comment]
Hi, apologies for not getting back with a capture file, I've just built the dev branch and can confirm I now see expected results, DHCPv4 creating the sessions and dhcpv6 reattempting and therefore successfully getting an IA prefix

Thank you so much, this will really help for testing in our lab and getting ipv6 deployed to all of our customers :-)

![image](https://user-images.githubusercontent.com/8687668/213560815-0d20aa52-7652-4197-9d8d-9cac8f173442.png)

**Comment by GIC-de** [comment]
I just released a new version 0.8.11 including this fix.
